recipe/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
     - backcall
     - colorama  # [win]
     - decorator
-    - jedi >=0.10
-    - pexpect  # [unix]
+    - jedi >=0.16
+    - pexpect >4.3  # [unix]
```

Comment:
@Carreau anything else i might have missed?

Reply:
No, I think that's good! You're awesome.